pkg/api/plugins.go

```diff
@@ -112,6 +112,10 @@ func (hs *HTTPServer) GetPluginList(c *models.ReqContext) Response {
 			Signature:     pluginDef.Signature,
 		}
 
+		if pluginDef.Errors != nil {
```

Comment:
It's generally recommended to check if a slice is empty rather than nil, since it will work both on empty and nil slices:
```suggestion
		if len(pluginDef.Errors) > 0 {
```

Reply:
This is a pointer so `len()` won't work here. Unless you believe this field type should change?

Reply:
@wbrowne Oh yeah, never mind. I couldn't tell the type from GitHub UI.